src/code/photo_album.asm

```diff
@@ -150,7 +150,8 @@ PhotoAlbumInit2Handler:
     call func_028_45CD                            ; $40DD: PhotoAlbumInit2Handler $CD $CD $45
     jp   PhotoAlbumIncrementState                 ; $40E0: PhotoAlbumInit2Handler $C3 $DB $44
 
-Data_028_40E3::
+; VRAM memory addresses for the top left corner of each photo
+PhotoAlbumMapLookupTable::
     db   $E1, $98, $E5, $98, $EC, $98, $F0, $98 ; $40E3 |........|
```

Comment:
For a future PR (if you feel inclined to), you could even turn these numbers into proper addresses :

```asm
    dw $98E1
    dw $98E5
    dw $88EC
    ; etc…
```